### PR TITLE
Edited the link provided in coupled_cluster.py

### PR DESCRIPTION
**Pull request by mamakancha**
I tried multiple times, but the link mentioned was not working So I have deleted the previous link. And included a new link to a lecture series on Coupled Cluster Theory.
Any suggestions to other good resources are appreciated :)

### CODE REVIEW

File: examples/intermediate/coupled_cluster.py [hunk 1]
```diff
@@ -44,7 +45,7 @@ def main():
     print("Calculates the Coupled-Cluster energy- and amplitude equations")
     print("See 'An Introduction to Coupled Cluster Theory' by")
     print("T. Daniel Crawford and Henry F. Schaefer III")
-    print("http://www.ccc.uga.edu/lec_top/cc/html/review.html")
+    print("Reference to a Lectur Series: http://vergil.chemistry.gatech.edu/notes/sahan-cc-2010.pdf")
```
Comment (jksuom) [inline examples/intermediate/coupled_cluster.py]: Typo: Lectur

Comment (mamakancha) [inline examples/intermediate/coupled_cluster.py]: Sorry for the typo. Corrected it.

File: examples/intermediate/coupled_cluster.py [hunk 2]
```diff
@@ -4,7 +4,8 @@
 Calculates the Coupled-Cluster energy- and amplitude equations
 See 'An Introduction to Coupled Cluster Theory' by
 T. Daniel Crawford and Henry F. Schaefer III.
-http://www.ccc.uga.edu/lec_top/cc/html/review.html
+
+Other Resource : Other Resource : http://vergil.chemistry.gatech.edu/notes/sahan-cc-2010.pdf
```
Comment (ranjithkumar007) [inline examples/intermediate/coupled_cluster.py]: Redundant ```Other Resource : ```  present here.

Comment (mamakancha) [inline examples/intermediate/coupled_cluster.py]: Thanks! Resolved!

**Comment by mamakancha** [comment]
@jksuom Suggested edits have been implemented. Please review.

**Comment by jksuom** [comment]
Thanks! This link may not be stable either, but it is working now.

**Pull request merged**